### PR TITLE
[DEV APPROVED] 10468 footer changes

### PR DESCRIPTION
**Pull request by andreferreiradlw**
[TP 10468](https://moneyadviceservice.tpondemand.com/RestUI/Board.aspx#page=board/5682182231901749200&appConfig=eyJhY2lkIjoiNTM0ODcyOThDQjBDMTMyMDgyMTBENEZEM0QwM0ZBQzkifQ==&boardPopup=userstory/10468/silent)

General styles for the secondary footer on the MAS website and blog imported to yeast in order to keep consistency between the two.

### CODE REVIEW

File: assets/layout/common/_footer.scss [hunk 1]
```diff
@@ -324,42 +324,127 @@
...
+
+.l-footer-secondary__list-item {
+  @include inline-block;
+  margin: $baseline-unit $baseline-unit $baseline-unit 0;
+  padding-right: $baseline-unit;
+  vertical-align: middle;
+
+  a {
+    // padding-right: $baseline-unit;
+    // margin: $baseline-unit $baseline-unit $baseline-unit 0;
```
Comment (davidtrussler) [inline assets/layout/common/_footer.scss]: These commented styles can be deleted

Comment (andreferreiradlw) [reply assets/layout/common/_footer.scss]: They're deleted now

File: assets/layout/common/_footer.scss [hunk 2]
```diff
@@ -324,42 +324,125 @@
...
+  
+  button {
+    background: transparent;
+    border: 0;
+    @include body(14, 24);
+    padding: 0;
+  }
+}
+
+
```
Comment (margOnline) [inline assets/layout/common/_footer.scss]: remove extra line

File: assets/layout/common/_footer.scss [hunk 3]
```diff
@@ -324,42 +324,125 @@
...
-    @include respond-to($mq-l) {
-      display: none;
-    }
-  }
 }
 
 .l-footer-secondary--inner {
   @include column(12);
+  @include respond-to($mq-l) {
+    display: -ms-flex;
```
Comment (margOnline) [inline assets/layout/common/_footer.scss]: 👍 for flex

File: assets/layout/common/_footer.scss (hunk 2 above)
Comment (andreferreiradlw) [reply assets/layout/common/_footer.scss]: These are now removed

**Pull request merged**